lib/portfinder.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * portfinder.js typescript definitions.
+ *
+ * (C) 2011, Charlie Robbins
+ */
+
+declare type PortfinderHandler = (err: Error, port: number) => void;
+
+/**
+ * The lowest port to begin any port search from.
+ */
+export declare let basePort: number;
+
+/**
+ * Responds with a unbound port on the current machine.
+ */
+export declare function getPort(callback: PortfinderHandler): void;
```

Comment:
Ok, thought that it was not "public" as it was not in the README.
The fact that the `port` option takes precedence over `basePort` has no implication in definition but I add it on comment for reference.